test/steps/data.go

```diff
@@ -46,6 +46,7 @@ func (data *Data) RegisterAllSteps(ctx *godog.ScenarioContext) {
 	registerKogitoBuildSteps(ctx, data)
 	registerKogitoRuntimeSteps(ctx, data)
 	registerKogitoDataIndexServiceSteps(ctx, data)
+	registerKogitoExplainabilityServiceSteps(ctx, data)
 	registerKogitoTrustyServiceSteps(ctx, data)
 	registerKogitoInfraSteps(ctx, data)
 	registerKogitoJobsServiceSteps(ctx, data)
```

Comment:
Can you add the new KogitoExplainabilityList custom resource in [this line](https://github.com/kiegroup/kogito-cloud-operator/blob/3fa1d80556ac5d89f26b4be17145196576863f19/test/steps/data.go#L112) as well? This is really useful to debug purposes after running the scenarios. 

I added the same comment in the PR master.